src/cita.ts

```diff
@@ -1,10 +1,10 @@
-import { Servicio } from "./servicio.ts";
+import { TipoServicio } from "./servicio.ts"
```

Comment:
Generalmente si haces una refactorización tienes o que indicarlo claramente en el cuerpo o hacerlo en un PR aparte. En este caso, entiendo que lo habéis mirado con los que han hecho esto, pero siempre debe quedar claro el ámbito del PR

Reply:
No era una refactorización como tal, lo he tenido que modificar porque daba un conflicto y no me dejaba hacer el commit :), pero lo tendré en cuenta